components/profile-section/profile-section.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styled from "styled-components";
+import Name from "./name";
+import RoleStyle from "./role";
+import SocialIcons from "./social-icons";
+
+const ProfileSectionStyle = styled.div`
+  display: flex;
+  flex-direction: column;
```

Comment:
You can delete flex here because the children's are div's and the default behaviour is a column layout.

Reply:
does not work on chrome

Reply:
children are spans.